reports/performance/_model.qmd

```diff
@@ -2,16 +2,26 @@
-- **Assessment Set**: This is the main model output for Valuations. It includes predictions on both seen (recently sold) and unseen (no recent sales) data. Sales are included for study if they're in the year prior to the assessment year i.e. 2023 assessments reference 2022 sales. This is the assessor standard way of measuring performance. This set should only be used to understand preliminary sales ratio outcomes.
+The following document describes performance for ```r metadata$run_id```, a ```r model_parameter_final$engine``` model built to re-assess the ```r metadata$assessment_triad``` triad in ```r metadata$assessment_year```. The document uses four sets of data to measure performance:
+
+- **Test Set**: A holdout set of data that the model didn't see during training. Per industry best practices, this set is used for model comparison and selection. For this run, the test set:
+  - Spanned from ```r m_test_min_date``` to ```r m_test_max_date```
+  - Contained ```r m_test_n_sales``` total sales, of which ```r m_test_n_sales_triad``` (```r m_test_n_sales_prop```) were in the ```r metadata$assessment_triad``` triad
+  - Had a median sale price of ```r m_test_med_sp```
+  - Included the last ```r m_test_split_prop``` of all training data sales
+- **Training Set**: Full data used to train to model for final prediction (prediction on the assessment set, not the test set). This _includes_ the test set, which is removed during model cross-validation and tuning. For this run, the training set:
+  - Spanned from ```r m_train_min_date``` to ```r m_train_max_date```
+  - Contained ```r m_train_n_sales``` total sales, of which ```r m_train_n_sales_triad``` (```r m_train_n_sales_prop```) were in the ```r metadata$assessment_triad``` triad
+  - Had a median sale price of ```r m_train_med_sp```
+  - Contained ```r m_train_n_outliers``` outliers (```r m_train_n_outliers_prop``` of the total sales)
+  - Used ```r metadata$model_predictor_all_count``` predictors, ```r metadata$model_predictor_categorical_count``` of which were categoricals
+- **Assessment Set**: This is the main model output for Valuations. It includes predictions on both seen (recently sold) and unseen (no recent sales) data. Sales are included for study if they're in the year prior to the assessment year i.e. 2023 assessments reference 2022 sales. This is the assessor standard way of measuring performance. This set should only be used to understand preliminary sales ratio outcomes. For this run, the assessment set:
+  - Contained sales from ```r m_assess_min_date``` to ```r m_assess_max_date``` for comparison
+  - Contained ```r m_assess_n_sales``` total sales for comparison
+  - Had a median sale price of ```r m_assess_med_sp```
+  - Had a lien/assessment date of ```r metadata$assessment_date```
+  - Used characteristics data from ```r metadata$assessment_data_year```
+  - Used ```r m_assess_stage_far``` and ```r m_assess_stage_near``` values for percentage change calculations
```

Comment:
Added descriptions and general information about the model and the datasets used to create and measure it.